app/client/components/payment/update/fieldWrapper.tsx

```diff
@@ -93,9 +96,9 @@ export class FieldWrapper extends React.Component<
         <div
           css={{
             border: `${
-              this.state.error?.message
-                ? "4px solid " + error[400]
-                : "2px solid " + neutral[60]
+              (this.state.error?.message && "4px solid " + error[400]) ||
+              (this.state.focus && "2px solid " + brand[500]) ||
+              "2px solid " + neutral[60]
```

Comment:
Might be clearer to move this logic elsewhere rather than doing it all inline? Not sure how we handle this kind of thing elsewhere though, so up to you.

Reply:
have added the correct focus state to the form fields